dev-ruby/vagrant_cloud/vagrant_cloud-3.0.2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby25 ruby26 ruby27"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_RECIPE_DOC="none"
+
+inherit ruby-fakegem
+
+DESCRIPTION="Vagrant Cloud API Library"
+HOMEPAGE="https://github.com/hashicorp/vagrant_cloud"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+ruby_add_rdepend "
+	>=dev-ruby/excon-0.73
+	>=dev-ruby/log4r-1.1.10
+"
+ruby_add_bdepend ">=dev-ruby/rake-12.3
+	test? (
+		>=dev-ruby/webmock-3.0
+	)
+"
+
+all_ruby_prepare() {
+	# # loosen dependencies
+	sed -e 'require_relative' \
+		-e '/excon/s/~>/>=/' \
```

Comment:
Why do you want to change this? That question applies to all of these changes, not just excon.

Reply:
It seems more logical to me, in the context of the distro to convert those require version to at least,
because that exact version may not be available and with at least it still can build *and* the information is still here,
if needed to be compared between ebuild and gemspec. (just my pov I can change it if you want)

Reply:
Upstream depends on that specific version and it can be hard to tell if that is for a good reason or not, so in general I like to keep this intact as much as possible, and adjust our slots accordingly.

Reply:
Ah, just realized that your change also means that future major versions will be valid now, and that is certain to break things.